DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -314,10 +317,10 @@ public void Configure(IApplicationBuilder app, IMigrationRunner migrationRunner,
 
         private Task RedirectToLogin(RedirectContext<CookieAuthenticationOptions> context)
         {
-            var applicationPath = new Uri(config.GetAppRootPath()).AbsolutePath.TrimEnd('/');
+            var applicationPath = new Uri(ConfigHelper.GetAppRootPath(config)).AbsolutePath.TrimEnd('/');

```

Comment:
I can see why this has had to change but this is also highlighting an issue in that we now have conflicts.
There is a ticket (HEEDLS-609) which is looking to address these issues I think.
Could you add a TODO here and reference the above ticket number please?